core/document-types/page-base.php

```diff
@@ -46,6 +46,9 @@ protected static function get_editor_panel_categories() {
 				'theme-elements-single' => [
 					'title' => esc_html__( 'Single', 'elementor' ),
 					'active' => false,
+					'promotion' => [
```

Comment:
Or is this the answer to the previous question?

Reply:
@hein-obox answered above.